pom.xml

```diff
@@ -4,18 +4,20 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
         http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-    <modelVersion>4.0.0</modelVersion>
+  <properties>
+	<bioformats.version>5.1.7</bioformats.version>
+    </properties>
 
     <parent>
         <groupId>ome</groupId>
         <artifactId>pom-omero-client</artifactId>
-        <version>5.2.1-SNAPSHOT</version>
+        <version>5.2.1</version>
```

Comment:
This can probably go to `5.2.4`: https://github.com/ome/pom-omero-client/commit/4463e48f3b7e4c36900930c6a7f7a768d0dcaac8#diff-600376dffeb79835ede4a0b285078036R9